### ISSUE TITLE
[Flake]: Nightly build exceeded table quota

### ISSUE THREAD

**Issue by coryan** [googleapis/google-cloud-cpp#3297]
We run a nightly build to test the Bigtable admin operations against production, as the quota for them is too small to run on each PR.  The build failed last night:

https://source.cloud.google.com/results/invocations/780ddc29-63be-4afc-ba85-e52bcc53f6a8

It seems like we are leaking tables with the pattern `tbl-${TIMESTAMP}-${RANDOM}` we should clean up any table older than N days before running the test.

**Comment by remyabel** [comment]
Description doesn't match the issue? grep `tbl-${TIMESTAMP}-${RANDOM}` doesn't return anything.

It looks like the actual issue is that the delete is not working, resulting in the leak:

```
Actual: DeleteTable(name=projects/cloud-cpp-testing-resources/instances/test-instance/tables/table-o65uxgtpg3ked8fgxgufos9etjx374t518vhv43su2je) Failed to read: projects/{936212892354}/instances/test-instance/tables/table-o65uxgtpg3ked8fgxgufos9etjx374t518vhv43su2je [NOT_FOUND]
```

**Comment by remyabel** [comment]
Disregard my last comment. I realize my error is that of course `DeleteTable` would fail because the table isn't created to begin with.

**Comment by devbww** [comment]
It looks like this is a duplicate of #3361, which was fixed by #3425.

@coryan ... do you concur?  [Aside: This is now out of SLO.]

**Comment by coryan** [comment]
Yes. Closing. We are also implementing the automatic table cleanups in the latest version of the examples.
